welcome.md

```diff
@@ -0,0 +1,110 @@
+Software developers as a rule are very particular about what kind of tools they like to use. And that makes perfect sense. Software development is a very nasty business if we didn't use software to make it easier to write software. Regardless of what type of programmer you will need to use:
+
+* [a decent editor](#decent-editor)
+* [version control](#version-control)
+* [package manager](#package-manager)
+* [test suite](#test-suite)
+
+## Decent editor
+Some say they cannot do without an IDE and some like their editor a bit more barebones. Then there is always a small amount of people who couldn't care less (@bertonens) about what kind of editor they use, as long is isn't slow.
+
+Editors you can use when doing Python/JavaScript in alphabetical order:
+* [Atom](https://atom.io/) - Atom is open source and free to use and hack on
+* `emacs` - is an open source command line text editor 
+* [PyCharm](https://www.jetbrains.com/pycharm/) - PyCharm has a free version, but you want the paid one
+* [Sublime Text](https://www.sublimetext.com/) - Sublime is free to try
+* `vim` - is an open source command line text editor 
+* [Visual Studio Code](https://code.visualstudio.com/) - VS Code is open source and free to use and hack on
+
+Get to know your editor of choice well. Use a cheatsheet to get acquainted with some tools it has. Look for a linter you like (a thing that checks if your code is nice and orderly), this will save you a ton of time.
+
+### Command line editors
+Command line editors are editors you can use from the terminal, locally or on the server. The above mentioned `vim` or `emacs` usually come pre-installed on most \*nix Operating Systems. If you really don't like using that on a server you can also use `nano`.
+
+## Package Manager
+Both platforms make it easy to maintain and release software without the need to download seperate packages, but using a package manager.
+
+JavaScript's defacto package manager is excellent and called [npm](https://npmjs.com). It is the largest collection language-specific software in the world and is packaged with [node](https://nodejs.org).
+
+Python uses [pip](https://pip.pypa.io/en/stable/installing/). You can use it in conjunction with virtualenvwrapper, which is a tool to seperate your packages important to your project from the rest of your system.
+
+# Learning JavaScript
+JavaScript has a superpower that no other language can compete with: it runs in every browser. Which means you can start learning right away without even downloading extra stuff.
+The other superpower JavaScript has is that it is slowly taking over the world. Almost anything you can think of has also been written for, ported to, or conceived in JavaScript, which is congruent with [Atwood's Law](http://blog.codinghorror.com/the-principle-of-least-power/). JavaScript does not only run in the browser, but you can also run JavaScript as a server side application or make it do a ton of crazy stuff with [node.js](https://nodejs.org/en/).
```

Comment:
Wij gebruiken python aan de backend omdat we python zijn gaan gebruiken aan
de backend en numpy/gdal etc. Python is niet inherent netter en leesbaarder
dan JavaScript.

On 18 March 2016 at 12:00, Reinout van Rees <notifications@github.com>
wrote:

> In welcome.md <https://github.com/nens/WOM/pull/3#discussion_r56640592>:
>
> > +
> > +Get to know your editor of choice well. Use a cheatsheet to get acquainted with some tools it has. Look for a linter you like (a thing that checks if your code is nice and orderly), this will save you a ton of time.
> > +
> > +### Command line editors
> > +Command line editors are editors you can use from the terminal, locally or on the server. The above mentioned `vim` or `emacs` usually come pre-installed on most \*nix Operating Systems. If you really don't like using that on a server you can also use `nano`.
> > +
> > +## Package Manager
> > +Both platforms make it easy to maintain and release software without the need to download seperate packages, but using a package manager.
> > +
> > +JavaScript's defacto package manager is excellent and called [npm](https://npmjs.com). It is the largest collection language-specific software in the world and is packaged with [node](https://nodejs.org).
> > +
> > +Python uses [pip](https://pip.pypa.io/en/stable/installing/). You can use it in conjunction with virtualenvwrapper, which is a tool to seperate your packages important to your project from the rest of your system.
> > +
> > +# Learning JavaScript
> > +JavaScript has a superpower that no other language can compete with: it runs in every browser. Which means you can start learning right away without even downloading extra stuff.
> > +The other superpower JavaScript has is that it is slowly taking over the world. Almost anything you can think of has also been written for, ported to, or conceived in JavaScript, which is congruent with [Atwood's Law](http://blog.codinghorror.com/the-principle-of-least-power/). JavaScript does not only run in the browser, but you can also run JavaScript as a server side application or make it do a ton of crazy stuff with [node.js](https://nodejs.org/en/).
>
> Maar wij gebruiken aan de serverkant alleen python omdat dat veel
> leesbaarder/netter is en vanwege de grote hoeveelheid libraries zoals numpy
> en gdal.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nens/WOM/pull/3/files/6f80f4beba457be7a762aaea5048436f9f554f4a#r56640592>
>



-- 
*Nelen & Schuurmans *

Postbus 1219  3500 BE Utrecht
Zakkendragershof 34-44 Utrecht
tel. 030 2330 200
@fritzvd <http://twitter.com/fritzvd>
www.nelen-schuurmans.nl